examples/clone.py

```diff
@@ -1,10 +1,11 @@
 """Clone.
 
-This trivial script demonstrates how to clone a remote repository.
+This trivial script demonstrates how to clone or lock a remote repository.
```

Comment:
huh, lock?

Reply:
I don't know haha, sorry, this is another merge request, but I accidentally committed it on the wrong branch, so this change can be ignored.

Reply:
https://github.com/dulwich/dulwich/pull/823